spring-cloud-stream-binders/spring-cloud-stream-binder-rabbit/src/main/java/org/springframework/cloud/stream/binder/rabbit/RabbitMessageChannelBinder.java

```diff
@@ -118,7 +117,7 @@
 
 	private static final int DEFAULT_PREFETCH_COUNT = 1;
 
-	private static final String DEFAULT_RABBIT_PREFIX = "xdbus.";
+	private static final String DEFAULT_RABBIT_PREFIX = "binder.rabbit.";
```

Comment:
why do we need to include "rabbit" here (this is the prefix for something that will be created on rabbit)